app/controllers/completeds_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CompletedsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @completeds = Completed.where(
+      'user_id = ? and created_at > ?',
+      current_user,
+      30.days.ago
+    )  end
```

Comment:
`end` at 9, 7 is not aligned with `def` at 4, 2<br>Unnecessary spacing detected.